coalaip/model_validators.py

```diff
@@ -78,7 +78,7 @@ def is_manifestation_model(instance, attribute, value):
                    "'{value}'").format(attr=attribute.name,
                                        cls=instance_name,
                                        value=manifestation_of)
-        raise ModelDataError(err_str)
+        print(err_str)
```

Comment:
If you want to output messages on the internal functioning of the module you should use `logging`.

`print` is to talk with the user, but this is a "passive" module.